src/transforms/remap.rs

```diff
@@ -69,12 +75,24 @@ impl Remap {
             timezone: config.timezone,
             drop_on_error: config.drop_on_error,
             drop_on_abort: config.drop_on_abort,
+            enrichment_tables,
         })
     }
 }
 
 impl FunctionTransform for Remap {
     fn transform(&mut self, output: &mut Vec<Event>, event: Event) {
+        let handle = self.enrichment_tables.handle();
```

Comment:
This is the problem with `evmap`.  To store the reader in Remap, it needs to store the `ReadHandleFactory`, since that can be passed between threads. But to read from it we need to get the `handle` here. This requires a lock, with removes any advantage of `evmap`.

Any suggestions welcome!